run_migration.py

```diff
@@ -1,4 +1,46 @@
-# TODO: make this dynamic so you can pass in the migration you want to run from the command line?
-from migrations import sample_timestamps
+from migrations import (
+  sample_timestamps,
+  update_mlwh_with_legacy_samples,
+)
+import sys
 
-sample_timestamps.run()
\ No newline at end of file
+##
+# Examples of how to run from command line:
+# python run_migration.py sample_timestamps
+# python run_migration.py update_mlwh_with_legacy_samples 200115_1200 200216_0900
+##
+
+print("Migration names:")
+print("* sample_timestamps")
+print("* update_mlwh_with_legacy_samples")
+
+def migration_sample_timestamps():
+    print("Running sample_timestamps migration")
+    sample_timestamps.run()
+
+def migration_update_mlwh_with_legacy_samples():
+    if not len(sys.argv) == 4:
+        print("Please add both start and end datetime arguments for this migration (format yymmdd_hhmm e.g. 200115_1200), aborting")
```

Comment:
I like the comments. Can we add info on whether or not the start and end dates are inclusive or exclusive?

Reply:
Added. And there's a new section in the Readme about how to run it too.